### DISCUSSION TITLE
How to use URL Scanner?

### DISCUSSION THREAD

**Discussion by HASNAINGAMERYT** [TrianguloY/URLCheck#289] [Q&A] [answered]
I've gotten the API Key of Virustotal, but don't know how to exactly insert it. Where am I supposed to put it?

**Comment by TrianguloY** [comment, marked as answer]
- Open the app
- Go to "Modules"
- Scroll and expand "URL Scanner"
- Paste the key in the "Api key" textbox.
- Enable the module (top right toggle, next to the "URL Scanner" text)

![Screenshot_20231127-125002.png](https://github.com/TrianguloY/UrlChecker/assets/10616202/195ba795-c85a-4e12-a10d-88b08dfdf024)

Next time you open a link you will have a Scan button to trigger it.

**Comment by HASNAINGAMERYT** [comment]
Idk how I missed that, but thanks a lot. It worked.
